velox/functions/prestosql/aggregates/tests/AggregationFunctionRegTest.cpp

```diff
@@ -54,6 +54,37 @@ TEST_F(AggregationFunctionRegTest, prefix) {
   });
 }
 
+TEST_F(AggregationFunctionRegTest, orderSensitive) {
+  // Remove all functions and check for no entries.
+  clearAndCheckRegistry();
+
+  std::set<std::string> nonOrderSensitiveFunctions = {
+      "sum",
+      "avg",
+      "min",
+      "max",
+      "count",
+      "arbitrary",
+      "bool_and",
+      "bool_or",
+      "every",
+      "checksum",
+      "count_if",
+      "geometric_mean",
+      "histogram",
+      "reduce_agg",
+      "any_value"};
+  aggregate::prestosql::registerAllAggregateFunctions();
+  exec::aggregateFunctions().withRLock([&](const auto& aggrFuncMap) {
+    for (const auto& entry : aggrFuncMap) {
+      if (!entry.second.metadata.orderSensitive) {
```

Comment:
nit: perhaps, verify that erase did remove an entry

```
if (!entry.second.metadata.orderSensitive) {
  ASSERT_EQ(1, nonOrderSensitiveFunctions.erase(entry.first));
}
```

Reply:
Done.